src/test/java/org/jboss/hal/testsuite/test/util/ConfigAreaChecker.java

```diff
@@ -145,16 +182,19 @@ public void invoke() {
             }
             fragment.saveAndAssert(expectedChange);
```

Comment:
I know that this controversy line is not caused by this PR but I belive that we should avoid and deprecate usage of asserts in UI elements. Mixture of UI and assert logic is neither clean nor safe (hiding failure reasons by Graphene proxies.) Could you please use 
```
fragment.save()
```
instead and extract assert logic here?